Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/mapping/CovidCertificateConfigMapper.kt

```diff
@@ -18,7 +18,9 @@ class CovidCertificateConfigMapper @Inject constructor() : CovidCertificateConfi
 
         return CovidCertificateConfigContainer(
             testCertificate = rawConfig.dgcParameters.mapCovidCertificateConfig(),
-            expirationThresholdInDays = rawConfig.dgcParameters.expirationThresholdInDays
+            expirationThresholdInDays = Duration.standardDays(
```

Comment:
Max said this is the number of date changes, not a multiple of 24h intervals. That's why I think this conversion is not accurate, however, it won't have any meaningful consequences for users.

Reply:
Seems like a duration to me though, whether we treat it as day changes would be up to the configs consumer. I'd prefer to model `expirationThresholdInDays` as a Duration.
I'll rename it `expirationThreshold` as whether it is hours/days/weeks is encoded in `Duration` and not something the config consumer has to care about.